concordium-node/src/dumper.rs

```diff
@@ -1,6 +1,6 @@
 //! Handles the `network_dump` feature.
 
-cfg_if! {
+cfg_if::cfg_if! {
```

Comment:
Is this intentional?

Reply:
Yes.

Reply:
> Looks good!
> 
> A CHANGELOG entry is currently missing.
> 
> I'm not sure why the ptrs to `concordium-base´ and `concordium-grpc-api´ are changed either? It seems unrelated to this PR? Is it at least on purpose? :)

Yes. Because they were not on main. And nothing that the node uses there has changed.

Reply:
> Is this intentional?

And the reason for the change is that this is the only remaining place where `cfg_if` is used. And as part of the cleanup related to the instrumentation flag I also removed this

```

#[macro_use]
extern crate cfg_if;

```

Which used to be needed (the macro_use) in older versions of Rust, but is no longer the way to do it for some years.